provider-syllabus/src/java/org/sakaiproject/archiver/provider/SyllabusArchiver.java

```diff
@@ -121,14 +121,22 @@ private void finaliseAttachmentsHtml(final SimpleSyllabus simpleSyllabus) {
 	}
 
 	/**
-	 * Surround the syllabus body string with html paragraph tags
+	 * Construct the syllabus html string
 	 *
-	 * @param syllabusBody
+	 * @param simpleSyllabus
 	 * @return syllabusBodyHtml
 	 */
-	private String getAsHtml(final String syllabusBody) {
+	private String getAsHtml(final SimpleSyllabus simpleSyllabus) {
+
+		final StringBuilder sb = new StringBuilder();
+
+		sb.append("<p>" + simpleSyllabus.getBody() + "</p>");
+
+		if (simpleSyllabus.getAttachments().length() > 0) {
```

Comment:
I presume simpleSyllabus.getAttachments() can't be null?

Reply:
No it starts off as ""